PaddleCV/Research/webvision2018/infer.py

```diff
@@ -0,0 +1,119 @@
+#copyright (c) 2019 PaddlePaddle Authors. All Rights Reserve.
```

Comment:
We only release inference code and model?

Reply:
yes